### ISSUE TITLE
Can fontawesomeicons have an alt property for SEO?

### ISSUE THREAD

**Issue by repejota** [vtalks/vtalks.net#71]
## Description

there is also something about role="link" aria-label="foo bar" that should be researched.

**Comment by repejota** [comment]
This issue was moved to vtalks/docs#12
